src/main/java/core/basesyntax/BollColors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum BollColors {
```

Comment:
enum class naming should be singular
```suggestion
public enum Color {
```